instructions.md

```diff
@@ -24,16 +24,16 @@ There are [57 tests](docs/CNFCertification-1.0-beta.md) included in the CNF Cert
 
 If you do not have a K8s cluster running, you can deploy your own cluster with [various tools](https://kubernetes.io/docs/setup/) or use a [public cloud or turnkey solution](https://kubernetes.io/docs/setup/production-environment/turnkey-solutions/)
 
-Kind is one option tested and supported by the CNF Test Suite team.  Follow the [kind install](https://github.com/cncf/cnf-testsuite/blob/main/KIND-INSTALL.md) instructions to setup a cluster in [kind](https://kind.sigs.k8s.io/)
+Kind is one option tested and supported by the CNTI Test Catalog team.  Follow the [kind install](https://github.com/cnti-testcatalog/testsuite/blob/main/KIND-INSTALL.md) instructions to setup a cluster in [kind](https://kind.sigs.k8s.io/)
 
 
 ## Running
-The standard tool for running these tests is the [CNF Test Suite](https://github.com/cncf/cnf-testsuite). 
+The standard tool for running these tests is the [CNTI Test Catalog](https://github.com/cnti-testcatalog/testsuite). 
 
-1. Download the [certification binary release](https://github.com/cncf/cnf-testsuite/releases/tag/v0.29.1) of the test suite
+1. Download the [certification binary release](https://github.com/cnti-testcatalog/testsuite/releases/tag/v0.29.1) of the test suite
```

Comment:
the binary release URL will need to be updated for CNTI Certification v2.0 - release number pending

Reply:
This should be updated along with 2.0 release changes in a dedicated PR.